db/migrate/20190701161400_add_parent_facility_code_index_to_intititutions.rb

```diff
@@ -0,0 +1,6 @@
+class AddParentFacilityCodeIndexToIntititutions < ActiveRecord::Migration
+  disable_ddl_transaction!
+  def change
+    add_index :institutions, [:version, :parent_facility_code_id] , using: :btree
```

Comment:
This needs `algorithm: :concurrently` too:
```suggestion
    add_index :institutions, [:version, :parent_facility_code_id], using: :btree, algorithm: :concurrently
```